2506.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @param {string[]} words
+ * @return {number}
+ */
+var similarPairs = function (words) {
+  let uniqArr = [];
+  uniqArr = words.map((word) => {
+    return [...new Set(word)].sort().join("");
+  });
```

Comment:
```suggestion
  const uniqArr = words.map((word) => {
    return [...new Set(word)].sort().join("");
  });
```
Use `const`.
You don't need to separate the declaration and 
 the initialization of `uniqArr`